internal/fs/udf/udf_test.go

```
@@ -276,3 +276,26 @@ func TestExtentReader_ReadsAcrossExtents(t *testing.T) {
 		t.Fatalf("data mismatch: got len=%d want len=%d", len(got), len(want))
 	}
 }
+
+func TestDirectoryEntriesSkipDeletedFIDs(t *testing.T) {
+	fid := func(chars byte, name string) []byte {
+		b := make([]byte, 38)
+		b[18] = chars
+		nameData := append([]byte{8}, name...)
+		b[19] = byte(len(nameData))
+		b = append(b, nameData...)
```

golangci-lint (2.12.2)

[error] append to slice `b` with non-zero initialized length

(makezero)